main.py

```diff
@@ -290,7 +291,7 @@ def get_board(self, access_token_in):
                                         requests.get(
                                             msg["data"]["name"],
                                             stream=True,
-                                            proxies=proxy.get_random_proxy(self),
+                                            proxies=proxy.get_random_proxy(self, name=None),
```

Comment:
This is going to always call get_random_proxy with name set to None. I don't think that;s what your intending to do

Reply:
That's for non worker calls so I did mean to put that there

Reply:
Why are you making the function call if it is always going to return the same value? That line of code won't perform anything

Reply:
my bad